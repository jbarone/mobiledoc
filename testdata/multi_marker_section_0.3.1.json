{
	"version": "0.3.1",
	"atoms": [],
	"cards": [],
	"markups": [
		["B"],
		["I"]
	],
	"sections": [
		[1, "P", [
				[0, [0], 0, "hello "],
				[0, [1], 0, "brave "],
				[0, [], 1, "new "],
				[0, [], 1, "world"]
			]
		]
	]
}
