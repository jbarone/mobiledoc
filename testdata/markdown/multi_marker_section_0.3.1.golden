**hello _brave new _world**

